File: Code coaches/Easy/Argentina/Argentina.swift
import Foundation

if  let input1 = readLine(),
    let ars = Decimal(string: input1), // Argentine peso
    let input2 = readLine(),
    let usd = Decimal(string: input2) // US dollar
{
	let arsToUsd: Decimal = 0.02
	let convertedUsd = ars * arsToUsd
	
	if convertedUsd > usd {
		print("Dollars")
	} else {
		print("Pesos")
	}
}

File: Code coaches/Easy/Argentina/Argentina.java
import java.math.BigDecimal;
import java.util.Scanner;

public class Argentina {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);

		BigDecimal ars = input.nextBigDecimal();
		BigDecimal usd = input.nextBigDecimal();

		BigDecimal arsToUsd = new BigDecimal("0.02");
		BigDecimal convertedUsd = ars.multiply(arsToUsd);

		if (convertedUsd.compareTo(usd) == 1) {
			System.out.println("Dollars");
		} else {
			System.out.println("Pesos");
		}

		input.close();
	}

}
